cli/tests/pcluster/config/test_section_cluster.py

```diff
@@ -253,6 +254,7 @@ def test_cluster_section_from_240_cfn(mocker, cfn_params_dict, expected_section_
                 "VolumeIOPS": "NONE,NONE,NONE,NONE,NONE",
                 "VolumeSize": "20,NONE,NONE,NONE,NONE",
                 "VolumeType": "gp2,NONE,NONE,NONE,NONE",
+                "Cores": "-1,-1",
```

Comment:
You should remove the "Cores" parameter from this test, because it is the `test_cluster_section_from_231_cfn` because it is testing a set of input parameters coming from a cluster created with the 2.3.1 version